src/main/resources/db/migration/V1_2__create_table_behandler_dialogmelding.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE BEHANDLER_DIALOGMELDING
+(
+    id            SERIAL PRIMARY KEY,
+    behandler_ref VARCHAR(50)  NOT NULL UNIQUE,
+    type          VARCHAR(30)  NOT NULL,
+    fnr           VARCHAR(11)  NOT NULL,
+    fornavn       VARCHAR(255) NOT NULL,
+    mellomnavn    VARCHAR(255),
+    etternavn     VARCHAR(255) NOT NULL,
+    partner_id    VARCHAR(50)  NOT NULL,
+    her_id        VARCHAR(50)  NOT NULL,
+    parent_her_id VARCHAR(50)  NOT NULL,
+    hpr_id        VARCHAR(50),
+    kontor        VARCHAR(255),
+    adresse       VARCHAR(255),
+    postnummer    VARCHAR(4),
+    poststed      VARCHAR(255),
+    orgnummer     VARCHAR(9),
+    created_at    timestamptz  NOT NULL,
+    updated_at    timestamptz  NOT NULL,
+    UNIQUE (partner_id, parent_her_id)
```

Comment:
Eller vil partner_id være unik på tvers av parent_her_id-er?

Reply:
Gitt støtte for ulike behandler typer:
Om man får orgnummer og ikke parent_her_id i en dialogmelding så vil den være null og da vil ikke dette fungere tror jeg(eller fungerer constrains på nullables?)

Reply:
Ok. Hvis parent_her_id er nullable så tror jeg ikke vi kan bruke unique her (eller vet du @geir-waagboe ?)
Og vi kan vel heller ikke ha en unique constraint på kun partner_id antar jeg.

Reply:
Jeg tror det går an å ha unique-constraint på en nullable-kolonne:
PostgreSQL treats NULL as distinct value, therefore, you can have multiple NULL values in a column with a UNIQUE index.